### ISSUE TITLE
it's not tensorflow code...

### ISSUE THREAD

**Issue by nextfeixiag** [takumayagi/fpl#1]

**Comment by nextfeixiag** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        So , why don't you provide the raw images?.....
        
    
        
            
        
    
    
        
        

        
    

    On 9/22/2018 14:21，Takuma Yagi<notifications@github.com> wrote： 


I'm afraid that we aren't planning to re-write the code with TensorFlow, sorry for your inconvenience.
Since Chainer is similar to PyTorch, I believe you can run without difficulties!
